frontend/src/components/form/ApiInput.vue

```diff
@@ -0,0 +1,155 @@
+<!--
+Displays a field as text or as an input field, depending on the editing prop.
+You can two-way bind to the value using v-model.
+-->
+
+<template>
+  <span>
+    <span v-if="!editing">{{ value }}</span>
+    <span v-if="editing">
+      <b-form
+        inline
+        class="mb-2">
+
+        <b-form-group
+          id="api-input-group"
+          :label="label"
+          label-for="api-input"
+
+          invalid-feedback="Dieses Feld kann nicht leer sein.">
+
+          <b-form-input
+            id="api-input"
+            v-model="$v.localValue.$model"
+            name="api-input"
+            class="mr-2 ml-2"
+            :state="required && $v.localValue.$dirty ? !$v.localValue.$error : null"
+            v-bind="$attrs" />
+
+        </b-form-group>
+
+        <b-button
+          v-if="!autoSave"
+          variant="primary"
+          @click="reset">
+
+          Reset
+        </b-button>
+
+        <b-button
```

Comment:
Ich verstehe nicht ganz den Sinn des Save-Buttons, sollten wir den nicht verstecken wenn `autoSave` aktiviert ist?

Reply:
😄 Ich auch nicht. Hab den zur convenience drin gelassen, weil er autoSave und success auch gerade mit anzeigt und für den User symbolisiert, dass das was gespeichert wird.

Ich denke, das wär generell ein gutes Thema für das nächste Meeting:
- Wo wollen wir AutoSave?
- Wo wollen wir, dass der User bewusst selbst speichert?
- Wie stellen wir Konsistenz sicher?
- Wie schaut die UI/UX aus (insbesondere bei AutoSave)?

https://ux.stackexchange.com/questions/12289/guidelines-for-when-to-auto-save-settings-vs-save-button
https://ux.stackexchange.com/questions/12741/disable-the-save-button-until-changes-are-made/12757#12757